homeassistant/components/victron_gx_mqtt/strings.json

```
@@ -0,0 +1,1335 @@
+      "battery_max_voltage_cell_id": {
+        "name": "Battery maximum voltage cell ID"
+      },
+      "battery_mid_voltage": {
+        "name": "DC Bus mid voltage"
+      },
+      "battery_mid_voltage_deviation": {
+        "name": "DC Bus mid voltage deviation"
+      },
+      "battery_min_cell_temperature": {
+        "name": "Battery minimum cell temperature"
+      },
+      "battery_min_cell_voltage": {
+        "name": "Battery minimum cell voltage"
+      },
+      "battery_min_temperature_cell_id": {
+        "name": "Battery minimum temperature cell ID"
+      },
+      "battery_min_voltage_cell_id": {
+        "name": "Battery minimum voltage cell ID"
+      },
+      "battery_nr_modules_blocking_charge": {
+        "name": "Battery number of modules blocking charge"
+      },
+      "battery_nr_modules_blocking_discharge": {
+        "name": "Battery number of modules blocking discharge"
+      },
+      "battery_nr_modules_offline": {
+        "name": "Battery number of modules offline"
+      },
+      "battery_nr_modules_online": {
+        "name": "Battery number of modules online"
+      },
+      "battery_power": {
+        "name": "Battery power"
+      },
+      "battery_soc": {
+        "name": "Battery charge"
+      },
+      "battery_soh": {
+        "name": "Battery state of health"
+      },
+      "battery_temperature": {
+        "name": "Battery temperature"
+      },
+      "battery_time_to_go": {
+        "name": "Battery time to go"
+      },
+      "battery_voltage": {
+        "name": "DC Bus voltage"
+      },
+      "charge_mode": {
+        "name": "Battery charge mode"
+      },
+      "dcload_current": {
+        "name": "DC load current"
+      },
+      "dcload_power": {
+        "name": "DC load power"
+      },
+      "dcload_voltage": {
+        "name": "DC load voltage"
+      },
+      "dcsystem_aux_voltage": {
+        "name": "DC System auxiliary voltage"
+      },
+      "dcsystem_current": {
+        "name": "DC System current"
+      },
+      "dcsystem_power": {
+        "name": "DC System power"
+      },
+      "dcsystem_voltage": {
+        "name": "DC System voltage"
+      },
+      "digitalinput_alarm": {
+        "name": "Digital input alarm"
+      },
+      "digitalinput_input_state_raw": {
+        "name": "Digital input raw state"
+      },
+      "digitalinput_state": {
+        "name": "Digital input state"
+      },
+      "digitalinput_type": {
+        "name": "Digital input type"
+      },
+      "evcharger_current": {
+        "name": "EV charger current"
+      },
+      "evcharger_mode": {
+        "name": "EV charger mode"
+      },
+      "evcharger_power_phase": {
+        "name": "EV charger power {phase}"
+      },
+      "evcharger_set_current": {
+        "name": "EV charger set current"
+      },
+      "evcharger_status": {
+        "name": "EV charger status"
+      },
+      "generator_gen_id_cool_down_timer": {
+        "name": "Generator cool down timer"
```

Hyphen like for "warm-up":

```suggestion
        "name": "Generator cool-down timer"
```